### ISSUE TITLE
IndexOutOfBoundsException

### ISSUE THREAD

**Issue by Drullkus** [TheCBProject/NotEnoughItems#127]
[Crashlog on pastebin](https://pastebin.com/QX21LmEz)

Hi, I received an `IndexOutOfBoundsException` while playing. I searched already and only found relevant issue #57 but that says it was fixed in NEI 181. I am on 220.

Not even sure how this happened, since I was trying to split a stack of clay blocks, in a regular chest gui. I see some NEI stuff in the stacktrace though.

**Comment by Drullkus** [comment]
That was September, I'm done with 1.10 forever lol

**Comment by covers1624** [comment]
Yeah figured, just got the time to clear the tracker, figured id give an actual explanation instead of just closing it :D
